### ISSUE TITLE
[Feature request] UI tweak: tooltip for lock icon, and expose its functionality in preferences

### ISSUE THREAD

**Issue by bokov** [lxqt/qterminal#855]
**Is your feature request related to a problem? Please describe.**

I accidentally must have clicked the lock button in the corner of the terminal that I never noticed before, thus disabling the hide-on-lost-focus behavior. From previous experience with Guake and YaKuake I expected to see that setting in the Preferences dialogue and spent a while looking for it.

**Describe the solution you'd like**

* A tooltip for the lock icon.
* Having that setting also be accessible through the Preferences dialogue, to reduce user surprise

**Describe eventual alternatives you've considered**

* Looking on here to see if it's a known bug :-)
* Installing Guake
* Finally noticing the lock button and face-palming

##### Context

In short, QTerminal behaves as intended, but this setting could be made a little more obvious.

**Comment by tsujan** [comment]
Which lock button? Please attach a screenshot of it.

**Comment by stefonarch** [comment]
QTerminal dropdown, right bottom, I suppose.

**Comment by tsujan** [comment]
> QTerminal dropdown, right bottom, I suppose.

There's no lock icon. Screenshot, please!

**Comment by stefonarch** [comment]
![schermata-09-16-15-53-01](https://user-images.githubusercontent.com/10681413/133624746-f210a64e-2e02-4f52-a469-b021630892d4.png)

**Comment by stefonarch** [comment]
Sure enough... I see it's where the tab bar is shown, but only for bottom or top - it's not displayed when right|left is set in preferences > tab bar and if tab bar isn't shown for one tab.

**Comment by tsujan** [comment]
> Sure enough... I see it's where the tab bar is shown

Oh, now I can see it :) I usually use the drop-down mode without tabs or tabbar.

@bokov is right. The button needs a tooltip. As he suggested, having an option in Preferences for its existence is a good idea.

Also, its absence with vertical tabbars seems like a small issue to me.

**Comment by tsujan** [comment]
A PR is here: https://github.com/lxqt/qterminal/pull/856

**Comment by stefonarch** [comment]
The tooltip should not end with a dot, no? (example in pcmanfm-qt: "go up")

**Comment by tsujan** [comment]
Agreed and done.
